packages/react-relay/relay-hooks/useLoadMoreFunction.js

```diff
@@ -201,7 +207,7 @@ function useLoadMoreFunction<TVariables: Variables>(
 
       // If the query needs an identifier value ('id' or similar) and one
       // was not explicitly provided, read it from the fragment data.
-      if (identifierField != null) {
+      if (identifierInfo != null) {
```

Comment:
I'm confused by the relationship between `identifierField` and `identifierInfo` in this scope. My sense right now is that they are actually both coming from the same metadata but that `identifierInfo` is extracted locally and `identifierField` is extracted by the parent function and passed in. Is that correct?

If not, maybe we could add a comment explaining the difference between the two? If not, maybe we could get rid of one of them and either always pass in a full `identifierInfo` object, or avoid the need for parent functions to pass it in, and instead always extract it locally?

Reply:
You're right, having both was redundant. I just changed it so that this function doesn't need parent functions to pass `identifierInfo`. Now we just extract it locally.